apps/bettafish/src/mixins.scss

```diff
@@ -6,7 +6,7 @@
     h4 { font-size: 1.25em; }
     h5 { font-size: 1em; font-weight: bold; text-transform: uppercase; }
     h6 { font-size: 1em; text-transform: uppercase; }
-    img { max-width: 100% }
+    img { max-width: 100%; display: inline; }
```

Comment:
I remember having some formatting issues around images that were either the _first_ or _last_ elements in a story or a long description. Have you tested those to make sure they don't do silly things?